src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -68,39 +70,43 @@ type ContactDetailsProps = {
 };
 
 const ContactDetails = ({ systemIntake }: ContactDetailsProps) => {
-  const {
-    id,
-    requestType,
-    requester,
-    businessOwner,
-    productManager,
-    isso,
-    governanceTeams
-  } = systemIntake;
+  const { id, requestType, requester, governanceTeams } = systemIntake;
   const formikRef = useRef<FormikProps<ContactDetailsForm>>(null);
   const { t } = useTranslation('intake');
   const history = useHistory();
+
+  const flags = useFlags();
+
   const [isReqAndBusOwnerSame, setReqAndBusOwnerSame] = useState(false);
   const [isReqAndProductManagerSame, setReqAndProductManagerSame] = useState(
     false
   );
 
+  const [
+    activeContact,
+    setActiveContact
+  ] = useState<SystemIntakeContactProps | null>(null);
+
+  const [
+    contacts,
+    { createContact, updateContact, deleteContact }
+  ] = useSystemIntakeContacts(id);
+
+  const requesterContact = useCedarContactLookup(
+    requester.name,
+    systemIntake.euaUserId
+  );
+
```

Comment:
This was failing for me on both local and okta authenticated, as well as connected to CEDAR VPN.  

Alternatively, another approach to get the current requester's data is to use the Okta hook, instead of a query to CEDAR

This may be more efficient and reliable than CEDAR

```
import { useOktaAuth } from '@okta/okta-react';

const { authState, oktaAuth } = useOktaAuth();

useEffect(() => {
    let isMounted = true;
    if (authState?.isAuthenticated) {
      oktaAuth.getUser().then((info: any) => {
        if (isMounted) {
          setIntakeRequester({
            commonName: info.name,
            euaUserId: info.preferred_username,
            email: info.email
          });
        }
      });
    }
    return () => {
      isMounted = false;
    };
  }, [authState, oktaAuth]);
```